vowpalwabbit/conditional_contextual_bandit.cc

```diff
@@ -489,6 +489,16 @@ void learn_or_predict(ccb& data, multi_learner& base, multi_ex& examples)
         save_action_scores(data, decision_scores);
         VW_DBG(examples) << "ccb "
                          << "slot:" << slot_id << " " << ccb_decision_to_string(data) << std::endl;
+        for (const auto& ex : data.cb_ex)
+        {
+          if (CB::ec_is_example_header(*ex)) { slot->num_features = (data.cb_ex.size() - 1) * ex->num_features; }
```

Comment:
I'm not sure any example will return true here if this gets run after shared_feature_merger, the previous PR made sense bc it was osn the finish path and that one gets run after shared_feature_merger reverts the merge but that shouldn't apply here in predict/learn